DetectorDescription/Detectors/src/SimpleParallelogram_geo.cxx

```diff
@@ -0,0 +1,65 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple parallelogram using dimensions to be used to define cone composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
+
+static Ref_t create_element(LCDD& lcdd, xml_h e, SensitiveDetector sens)  {
+
+
+  xml_det_t    x_det = e;
+  string       name  = x_det.nameStr();
+  DetElement   parallelogramDet( name, x_det.id() );
+  
+  Volume experimentalHall =  lcdd.pickMotherVolume( parallelogramDet ) ;
+  
+  xml_comp_t  parallelogramDim ( x_det.child( _U(dimensions) ) );  
+  ///correct implementation but bug in the xml parser??
+
+   const double parallelogramPoints[16] = {parallelogramDim.x0(),parallelogramDim.y0(),
```

Comment:
The constructor doesn't like it very much...
it expects:
DD4hep::Geometry::EightPointSolid::EightPointSolid(double, const double*)